tests/src/Common/Platform/platformdefines.h

```diff
@@ -55,6 +55,11 @@ typedef const WCHAR *LPCWSTR, *PCWSTR;
 #define __stdcall
 #define _cdecl
 #endif
+#ifndef CROSS_MSC_VER_DEFINES
+#define CROSS_MSC_VER_DEFINES
+#define __FUNCSIG__ "<clang-3.5 signature not supported>"
+#define __int64 long long
+#endif // CROSS_MSC_VER_DEFINES
```

Comment:
@rartemev I've asked before, but since you haven't answered it yet and this discussion is related, let me ask again. How come the native parts of the tests don't build now? They were building on Unix before just fine - to run CoreCLR tests on Unix, we were copying the managed parts from Windows and native parts were built on Unix as part of the CoreCLR native build.